src/plugins/rovpilot-wire/index.js

```diff
@@ -44,7 +44,7 @@
     deps.globalEventLoop.on('mcu.status', function (status) {
       if ('targetDepth' in status) {
         self.state.depth.enabled = status.targetDepth != DISABLED;
-        self.state.depth.targetDepth = Number(status.targetDepth) / 100;
+        self.state.depth.targetDepth = Number(status.targetDepth);
```

Comment:
As long as the firmware has been updated to match passing meters as floats along with the node side messaging then things should be fine. Just validating that the changes were made in all places that it needed to be.